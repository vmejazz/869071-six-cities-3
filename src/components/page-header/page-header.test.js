import React from "react";
import renderer from "react-test-renderer";
import PageHeader from "./page-header.jsx";
import customHistory from "../../history.js";
import {Router} from "react-router-dom";
jest.mock(`../user-profile/user-profile.jsx`, () => `UserProfile`);

it(`<PageHeader /> should render `, () => {
  const tree = renderer
      .create(
          <Router history={customHistory}>
            <PageHeader
            />
          </Router>
      )
      .toJSON();

  expect(tree).toMatchSnapshot();
});
