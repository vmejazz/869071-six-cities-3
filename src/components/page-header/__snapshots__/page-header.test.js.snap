// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeader /> should render  1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <a
          className="header__logo-link header__logo-link--active"
          href="/"
          onClick={[Function]}
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height="41"
            src="/img/logo.svg"
            width="81"
          />
        </a>
      </div>
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <li
            className="header__nav-item user"
          >
            <UserProfile />
          </li>
        </ul>
      </nav>
    </div>
  </div>
</header>
`;
